docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
```

Comment:
it's not clear how headless and CI differ

Reply:
I will replace the references from CI to headless.